exported_data_bucket.tf

```diff
@@ -1,24 +1,41 @@
+# This bucket is used to store a JSON file containing all exported
+# findings data to be imported into our AWS database.
```

Comment:
I think this is a bit more accurate, don't you?
```suggestion
# This bucket is used to store JSON files containing exported
# assessment findings data to be imported into our AWS database.
```

Reply:
Please see [a5e0343](https://github.com/cisagov/findings-data-import-terraform/pull/36/commits/a5e0343952066821f98ec6a0a99d140fd4442d28).